server/api/views.py

```
@@ -37,7 +38,27 @@ def create(self, request, *args, **kwargs):
             status=status.HTTP_201_CREATED
         )
-
+    
+    def get(self, request, *args, **kwargs):
+        """
+        Handles a user search which returns the top 10 users based on the
+        search term (such as users with a similar name). Searches the username,
+        display_name, and email fields in a case insentitive way.
+        """
+        data = request.data
+        users = User.objects.filter(
```

Ruff (0.8.2)

Undefined name `User`

(F821)